spidermon/contrib/monitors/mixins/validation.py

```diff
@@ -84,125 +84,259 @@ def __init__(self, stats, prefix=None):
 
 
 class ValidationMonitorMixin(StatsMonitorMixin):
+    def __init__(self, new_behavior=False):
+        super(ValidationMonitorMixin, self).__init__()
+        self.new_behavior = new_behavior  # how to handle empty field lists
```

Comment:
Maybe a better name here would avoid the comment and make the `if` statements below more readable